content/concepts/transports/webrtc.md

```diff
@@ -101,17 +101,85 @@ Authenticity is achieved by succeeding the
+However, browsers impose certain restrictions on application code (such as libp2p browser nodes).
+Applications are sandboxed and face constraints on security and networking.
+For instance, browsers do not permit direct access to raw network sockets.
+Additionally, it's a sure bet that libp2p browser nodes will be behind a NAT/firewall.
+Due to these restrictions, browser nodes cannot listen for incoming connections,
+and as a result, they cannot communicate with other browser nodes.
 
-The technical specification and initial implementations of WebRTC
-Browser-to-Browser connectivity is planned for release in early 2023.
-Track the progress [here](https://github.com/libp2p/specs/issues/475).
+Thankfully, libp2p solves this problem and enables browser node to browser node connectivity by supporting a transport called WebRTC W3C.
+
+{{< alert icon="" context="info">}}
+Due to the restrictions mentioned above, libp2p browser nodes are private nodes (meaning they cannot be dialed from outside of their local network).
+However, in principle, this use case can be thought of not just as browser-to-browser but private node to private node.
+This WebRTC solution enables connectivity between two non browser nodes behind a NAT/firewall as well
+{{< /alert >}}
+
+#### Transport Internals
+
+The libp2p WebRTC W3C transport is enabled by supporting the [W3C defined](https://w3c.github.io/webrtc-pc/#introduction) `RTCPeerConnection` API.
+This core API enables p2p connectivity and provides methods for establishing connections and transferring streams of data between peers.
+Running instances of libp2p that support this transport will have `/webrtc-w3c` in their multiaddr.
+
+However, there's more to p2p connections that what `RTCPeerConnection` provides. Crucially, signaling isn't built into the WebRTC API.
+{{< alert icon="" context="info">}}
+Signaling is the process of coordinating communication and exchanging metadata about the communication (i.e. initializing, closing, or reporting errors about connections).
+{{< /alert >}}
+
+For this transport, libp2p supports its own signaling protocol which has the protocol id: `webrtc-w3c-signaling`.
+
+#### How browser-to-browser connectivity works
 
 <!-- TO ADD DIAGRAM -->
 
+Suppose we have three network entities:
+
+- _Node A_ - a libp2p node running in the browser
+- _Node B_ - another browser libp2p node running on a different browser instance
+- _Relay R_ - a libp2p relay node
+
+In this connectivity scenario, _A_ wants to connect to _B_.
+This roughly works as follows:
+
+- _A_ and _B_ are both connected to _R_
+- _B_ appends `webrtc-w3c` to its multiaddress and that multiaddress is relayed by _R_.
+- _A_ will discover _B_'s multiaddress via _R_
+- _A_ sees that _B_ supports `webrtc-w3c` and establishes a relayed connection to _B_
+- Over the relayed connection, _A_ will:
+  - Create an _outbound_ `RTCPeerConnection`
```

Comment:
Just to avoid confusion, one does not create an `RTCPeerConnection` over the relayed connection, but rather with the help of the relayed connection. Is the phrasing only confusing me?